src/ui/common/src/components/integrations/cards/text.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import React from 'react';
+
+// Use in place of Typography when you need the text to be truncated with an ellipsis.
+export const TruncatedText = styled(Typography)(() => {
+  return {
+    fontFamily: 'Roboto',
```

Comment:
should be able to leave the fontface alone since the fontFace that MUI uses is Roboto.

Reply:
yeah I actually got rid of this in the latest commit after some digging. Same issue though